src/status_im/contexts/profile/settings/view.cljs

```diff
@@ -26,7 +26,7 @@
     {:list-type       :settings
      :container-style {:padding-bottom 12}
      :blur?           true
-     :data            (filter show-settings-item? data)}]))
+     :data            (doall (filter show-settings-item? data))}]))
```

Comment:
Because the one of the settings items uses a re-frame subscription, we're warned to not use a lazy-sequence, so we use `doall` to realise the lazy sequence. Alternatively we could also use `filterv`.